### ISSUE TITLE
Published Minified Code `sourceMappingURL` Incorrect for Versions 6.12.2+

### ISSUE THREAD

**Issue by chucknelson** [systemjs/systemjs#2421]
The published minified code `sourceMappingURL` for SystemJS versions `6.12.2+` seem to be incorrect.

### Expected Behavior

`sourceMappingURL` location should be something like:

`dist/system.min.js`
```
//# sourceMappingURL=system.min.js.map
```

### Actual Behavior

`sourceMappingURL` locations for minified code start looking like this starting at `6.12.2+`

`dist/system.min.js`
```
// 6.12.2
//# sourceMappingURL=##.min.js.map

// 6.12.3+
//# sourceMappingURL=system.min.js
```

### References

- https://cdn.jsdelivr.net/npm/systemjs@6.12.1/dist/system.min.js
- https://cdn.jsdelivr.net/npm/systemjs@6.12.2/dist/system.min.js
- https://cdn.jsdelivr.net/npm/systemjs@6.12.3/dist/system.min.js

**Comment by guybedford** [comment]
Yes, make sure to use 6.12.4.

**Comment by chucknelson** [comment]
> Yes, make sure to use 6.12.4.

Also seems to have the same issue. 

//# sourceMappingURL=system.min.js

See https://cdn.jsdelivr.net/npm/systemjs@6.12.4/dist/system.min.js

**Comment by wenerme** [comment]
This issues https://github.com/systemjs/systemjs/issues/2420 come to a full circle and back to systemjs 😆

**Comment by guybedford** [comment]
This should now be fixed in 6.12.6.
